### PR TITLE
Add new Windows build recipe

### PR DESCRIPTION
**Pull request by brian-rose**
This is closely following the recipe in [pyspharm-feedstock](https://github.com/conda-forge/pyspharm-feedstock), using mingwpy.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,12 +12,13 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [win or not py2k]
+  number: 2
+  skip: True  # [win and py3k or win64 or not py2k]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: When you change things that will affect the build matrix, like the `skip` section, you need to rerender to get the change into the CIs configuration. See http://conda-forge.github.io/docs/conda_smithy.html

Comment (brian-rose) [reply recipe/meta.yaml]: Thanks! Slowly figuring all this stuff out...

Comment (ocefpaf) [reply recipe/meta.yaml]: Don't worry. We have a lot of moving parts. Feedback on how to improve is welcomed :wink:

File: recipe/meta.yaml [hunk 2]
```diff
@@ -12,12 +12,13 @@ source:
...
+  number: 3
+  skip: True  # [win and py3k or win64 or not py2k]
 
 requirements:
   build:
     - toolchain
+    - mingwpy  # [win]
     - gcc  # [osx]
     - python
     - setuptools
```
Comment (ocefpaf) [inline recipe/meta.yaml:24]: You will need to add a pre-processor selector to eliminate `libgcc` on Windows.

**Comment by brian-rose** [comment]
Why is the check not running on Appveyor anymore?

**Comment by ocefpaf** [comment]
Sometimes we are rate limited. Waiting a bit, closing and opening the PR should do be enough to restart AppVeyor.

**Comment by brian-rose** [comment]
The Windows build passes! So cool! Thanks so much @ocefpaf for pointing me to the recipe for [pyspharm](https://github.com/conda-forge/pyspharm-feedstock).

**Comment by brian-rose** [comment]
Will merge this now and then see about extending the build to win64.

**Pull request merged**